data_warehouse/views.py

```diff
@@ -28,3 +37,33 @@ def get_canvas_options(self, request):
         '''getting canvas-options'''
         data = CanvasOptionsService.get_canvas_options()
         return Response(data)
+
+    @action(detail=False, methods=['GET'], url_path='table-export',
+            url_name='table-export')
+    def export(self, request):
+        '''Return a xls file stream.'''
+        if 'id' not in request.GET:
```

Comment:
Please change `id` to another name, FYI, `table_type`, `id` makes me think of this is an instance in the database.